SwiftGen.playground/Pages/Storyboards-Demo.xcplaygroundpage/Contents.swift

```diff
@@ -117,6 +117,4 @@ segues for a specific storyboard.
 //  case .Custom:
 //    // Prepare for your custom segue transition
 //  case .NonCustom:
-//    // Pass in information to the destination View Controller
-//  }
-//}
+//    // Pass in information to the destinatio
```

Comment:
No idea. Let me look. I didn't touch the playground until just now with these comments.